<ui version="4.0" >
 <class>ReplaceDialog</class>
 <widget class="QDialog" name="ReplaceDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Replace</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Replace:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="searchText" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>With:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="replaceText" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QCheckBox" name="checkBox_wrap" >
         <property name="text" >
          <string>Wrap around</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_backwards" >
         <property name="text" >
          <string>Find backwards</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_selection" >
         <property name="text" >
          <string>Search within selection</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QCheckBox" name="checkBox_case" >
         <property name="text" >
          <string>Case sensitive</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_words" >
         <property name="text" >
          <string>Whole words</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_regex" >
         <property name="text" >
          <string>Regular expression</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="bottomMargin" >
        <number>3</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="regexStatus" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>127</red>
               <green>127</green>
               <blue>127</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="1" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Abort|QDialogButtonBox::NoButton|QDialogButtonBox::Ok|QDialogButtonBox::SaveAll</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>searchText</tabstop>
  <tabstop>replaceText</tabstop>
  <tabstop>checkBox_wrap</tabstop>
  <tabstop>checkBox_backwards</tabstop>
  <tabstop>checkBox_selection</tabstop>
  <tabstop>checkBox_case</tabstop>
  <tabstop>checkBox_words</tabstop>
  <tabstop>checkBox_regex</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
